src/epd2in9_v2/mod.rs

```diff
@@ -227,6 +227,7 @@ where
         Ok(())
     }
 
+    /// actually is the "Turn on Display" sequence
```

Comment:
Is waveshare using a different sequence for the update?

Reply:
Their code calls it like that. Also using this method makes the screen flicker, seems to be more than just updating, yes.

Reply:
Yeah the flicker is kind of normal to prevent ghosting (see https://electronics.stackexchange.com/questions/20276/why-does-flashing-prevent-ghosting-on-e-ink-displays for more infos)

Reply:
I have some little ghosting but I am fine with it doing this for fast updates:

```rust
        cur_epd
            .update_old_frame(&mut spi, empty_display.buffer(), &mut delay)
            .unwrap();
        cur_epd
            .update_new_frame(&mut spi, display.buffer(), &mut delay)
            .unwrap();
        cur_epd.display_new_frame(&mut spi, &mut delay).unwrap();
```

that mimicks also what their example C-code does in their "GUI" to update the icons. It however does the turn-on-sequence every 100th redraw or when the display fundamentally changes.